app/models/document.rb

```diff
@@ -45,7 +45,9 @@ class Document < ActiveRecord::Base
                           end)
 
 
-  acts_as_searchable :columns => ['title', "#{table_name}.description"], :include => :project
+  acts_as_searchable :columns => ['title', "#{table_name}.description"],
+                    :include => :project,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.